api/all.yaml

```diff
@@ -2303,6 +2535,214 @@ components:
         metastore_id:
           description: Unique identifier of the metastore.
           type: string
+    StorageCredentialInfo:
+      type: object
+      properties:
+        name:
+          description: The credential name. The name must be unique within the metastore.
+          type: string
+        aws_iam_role:
```

Comment:
cant you give descriptions here regarding what this is?
same qs for azure below